components/Task.js

```diff
@@ -49,15 +52,17 @@ const TaskWrap = styled.ul`
   background: rgba(255, 255, 255, 0.4);
   box-shadow: 0px 4px 20px rgba(0, 0, 0, 0.1);
   border-radius: 8px;
-  margin: 1.5rem;
+  margin-top: 1.5rem;
+  margin-left: 1.5rem;
+  margin-right: 1.5rem;
   padding-bottom: 0.6rem;
 `;
 
-const TaskTitle = styled.h3`
+const TaskTitle = styled.li`
   font-family: var(--font-primary);
   font-style: normal;
   font-weight: 400;
   font-size: 1.2rem;
-  padding-left: 0.8rem;
-  padding-top: 0.6rem;
+  padding-left: 1rem;
+  padding-top: 1rem;
```

Comment:
instead of using padding left and top on the title, you can use the padding in the Taskwrap, as well as turning it into a grid with a common gap of some amount. in that way, the parent element takes care of assigning the space for the elements it holds.